tests/utils.py

```diff
@@ -557,6 +559,8 @@ def to_python_requires(
     config_file: str = os.path.join(dest_dir, module.ModuleV2.MODULE_FILE)
     config: configparser.ConfigParser = configparser.ConfigParser()
     config.read(config_file)
+    if use_four_digits:
+        config["metadata"]["use_four_digits"] = "True"
```

Comment:
Why not ?
```suggestion
        config["metadata"]["use_four_digits"] = True
```

Reply:
I think I tried it and this will be written to the .cfg and it only accepts strings